src/main/java/com/cloudlabs/server/compute/ComputeController.java

```diff
@@ -23,39 +23,21 @@ public class ComputeController {
 
 	// Create a new public instance with the provided "instanceName" value in the specified
 	// project and zone.
-	@PostMapping("/create")
-	public String create(@RequestBody JsonNode request)
+	
+	@DeleteMapping("/{project}/{zone}/{instanceName}")
+	public String delete((@PathVariable String project, @PathVariable String zone, @PathVariable String instanceName))
 			throws IOException, InterruptedException, ExecutionException, TimeoutException {
 
 		try {
-			String machineType = String.format("zones/asia-southeast1-b/machineTypes/%s",
-					request.get("selectedInstanceType").get("name").asText());
-			String sourceImage = String
-					.format("%s%s", request.get("selectedImage").get("project").asText(),
-							request.get("selectedImage").get("name").asText());
-			long diskSizeGb = 10L;
-			String networkName = "default";
-			String instanceName = request.get("name").asText();
-			String startupScript = request.get("script").asText();
+        	boolean deletionResult = computeService.deleteInstance(project, zone, instanceName);
```

Comment:
Project and zone should be environment variables, and not retrieved from the request.